Mathlib/Tactic/Linarith/Preprocessing.lean

```diff
@@ -134,8 +134,6 @@ def mk_natCast_nonneg_prf (p : Expr × Expr) : MetaM (Option Expr) :=
 def Expr.Ord : Ord Expr :=
 ⟨fun a b => if Expr.lt a b then .lt else if a.equal b then .eq else .gt⟩
```

Comment:
This is no longer needed, right?  (It was being used before to compare the expressions for the atoms, IIUC.)

Reply:
Correct; should I deprecate it?

Reply:
Yep, or delete!